src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala

```diff
@@ -179,11 +192,11 @@ final class ScalaParser(logger: Logger) {
         IntRef
       case "Long" =>
         LongRef
-      case "Double" =>
+      case "Double" | "BigDecimal" =>
```

Comment:
That's distinct from enum support (and mixing bigdec and double can be an issue)

Reply:
Hmm there isn't much in TypeScript out of the box other than `number` that can represent `BigDecimal`s... what would you suggest than? Maybe just introduce a separate ScalaModel distinct from `DoubleRef` (like `BigDecimalRef`) that would convert to TS `number` anyway after Scala->TS compilation?

Reply:
Anyway that should be discussed separately from this PR

Reply:
@cchantep please see #38

Reply:
Removed the related code from this PR (on 2nd round of fixes commit)